src/Core/Checkout/Cart/Facade/Traits/ItemsIteratorTrait.php

```diff
@@ -34,11 +34,12 @@ public function getIterator(): \ArrayIterator
             };
         }
 
+        /**
```

Comment:
Should the method return type not just be `ContainerFacade` then ?

Reply:
no, that won't work. Ondrey explained it here https://github.com/phpstan/phpstan/discussions/12727

it will be unified to ItemFacade, as the ContainerFacade is extending ItemFacade. 

currently this is intended behaviour, but there is also an issue to improve on that